ui-components/src/global/vendor/AgPagination.tsx

```diff
@@ -45,33 +47,15 @@ const AgPagination = ({
     dataPagerClassName.push("sticky")
   }
 
+  const paginationBarClassName = ["pagination-bar"]
```

Comment:
nit: I might name this `data-pager` and rename the current `data-pager` to `data-pager-container` or something similar.

Reply:
Oh nice, I like that - updated!